source/docs/software/advanced-controls/introduction/control-system-basics.rst

```diff
@@ -1,10 +1,21 @@
 Control System Basics
 =====================
 
-.. note:: This article is taken out of `Controls Engineering in FRC <https://file.tavsys.net/control/controls-engineering-in-frc.pdf>`__ by Tyler Veness with permission.
+.. note:: This article includes sections of `Controls Engineering in FRC <https://file.tavsys.net/control/controls-engineering-in-frc.pdf>`__ by Tyler Veness with permission.
+
+The Need for Control Systems
+----------------------------
 
 Control systems are all around us and we interact with them daily. A small list of ones you may have seen includes heaters and air conditioners with thermostats, cruise control and the anti-lock braking system (ABS) on automobiles, and fan speed modulation on modern laptops. Control systems monitor or control the behavior of systems like these and may consist of humans controlling them directly (manual control), or of only machines (automatic control).
 
+The thing that is common to all of these things: They all have a mechanism which does useful work, but cannot be *directly* commanded to the state that is desired.
```

Comment:
Remove "the thing that is."  Reads better as just "Common to all of these things: a mechanism which does useful work, but cannot be *directly* commanded to the state that is desired.